src/utils/BatchManager.js

```diff
@@ -1,5 +1,8 @@
-function errorToSerializable(err) {
-  if (!err) return null;
+function errorToSerializable(error) {
+  if (error === undefined) return null;
```

Comment:
why return null here, even on `undefined`? is this function ever called when it's not actually an error?